src/components/Footer/index.js

```diff
@@ -27,9 +27,11 @@ const Footer = ({ as, direction, ...rest }) => {
     <FooterContainer fluid as="footer" {...rest}>
       <ContactPanelRow>
         <ContactPanelColumn sizes={{ xs: 12, md: 4 }} direction="row">
-          <WebChatImg width="60px" height="60px" alignSelf="start" />
+          <Heading level={5}>
+            <WebChatImg width="60px" height="60px" alignSelf="start" />
+          </Heading>
```

Comment:
What I had in mind here was to replicate the existing RAD markup, whereby the icon is nested inside the heading. I was thinking along the lines of: 
```
<Heading level={2}>
  <WebChatImg/>
  Web Chat
</Heading>
```
which I believe would make it possible to render the two different variations of the layout that exist on RAD.